.github/workflows/test.yml

```diff
@@ -0,0 +1,73 @@
+name:
```

Comment:
This  file is not necessary?
you should merge with master

Reply:
this file is action for test site, will only be runned after kun-dev updated